webapp/greenhouse_api.py

```diff
@@ -137,6 +138,21 @@ def submit_to_greenhouse(form_data, form_files, job_id="1658196"):
     return response
 
 
+def parse_feed_department(feed_department):
+    if (
```

Comment:
I kind of find this hard to read, maybe a `switch` or a `map` would be more appropriate.

Reply:
Done